frontend/src/components/RisModForm.vue

```diff
@@ -353,13 +353,20 @@ const selectableAknElementsEventHandlers = Object.fromEntries(
     class="grid h-full max-h-full grid-cols-1 grid-rows-[min-content,min-content,1fr,min-content] gap-y-12 overflow-auto"
   >
     <div class="grid grid-cols-2 gap-x-16">
-      <RisDropdownInput
-        id="timeBoundaries"
-        v-model="selectedElement"
-        label="Zeitgrenze"
-        :items="timeBoundaries"
-        @change="$emit('generate-preview')"
-      />
+      <div class="flex flex-col gap-6">
+        <!-- eslint-disable-next-line vuejs-accessibility/label-has-for -->
+        <label id="timeBoundariesLabel" class="ris-label2-regular"
```

Comment:
Todo: should not use a hardcoded ID, use `useElementId` instead. See [styleguide](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Avoid-using-IDs.-Do-not-assume-them-to-be-human-readable-or-static.)